src/dmd/identifier.d

```diff
@@ -138,18 +137,57 @@ nothrow:
 
     private extern (D) __gshared StringTable!Identifier stringtable;
 
+    /**
+     * Generates a new identifier.
+     *
+     * Params:
+     *  prefix = this will be the prefix of the name of the identifier. For debugging
+     *      purpose.
+     */
     extern(D) static Identifier generateId(const(char)[] prefix)
+    {
+        return generateId(prefix, newSuffix, false);
+    }
+
+    /**
+     * Generates a new anonymous identifier.
+     *
+     * Params:
+     *  name = this will be part of the name of the identifier. For debugging
+     *      purpose.
+     */
+    extern(D) static Identifier generateAnonymousId(const(char)[] name)
+    {
+        return generateId("__anon" ~ name, newSuffix, true);
+    }
+
+    /**
+     * Generates a new identifier.
+     *
+     * Params:
+     *  prefix = this will be the prefix of the name of the identifier. For debugging
+     *      purpose.
+     *  suffix = this will be the suffix of the name of the identifier. This is
+     *      what makes the identifier unique
+     */
+    extern(D) static Identifier generateId(const(char)[] prefix, size_t suffix)
```

Comment:
Would it be possible to re-add this back to `extern(C++)`?  I was bitten by the removal in #10540 and have had to work around it ever since.

Reply:
Why didn't you just add an overload?

Reply:
Done.

Reply:
> Why didn't you just add an overload?

Meh, I have no time or will power to maintain a fork of the front-end. :-)